### PR TITLE
fix(Milestone):Fixed Milestone Add Task

### PR DESCRIPTION
**Pull request by sanjaykrishna1203**
### Functionality:
Fixed Milestone Component

### Solution:
- Fixed Milestone Add Task
- Fixed Task Data refresh on Create new Task

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
Try Adding tasks to milestone, check whether it is getting added.
Create a new task and check whether it is visible in task tracker

### CODE REVIEW

File: functions/model/milestone/tark/addTask.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+ /**
+ * Description
+ * @param {any} request
+ * @param {any} response
+ * @return {any}
+ */
+exports.addTaskToMilestone = function(request,response){
+    console.log("dsfkjsadnfkjanskjdfnkjn");
```
Comment (Pendyala-thanvi) [inline functions/model/milestone/tark/addTask.js:36]: Remove the console

File: functions/model/milestone/tark/addTask.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    console.log( request.body.data);
+    const uid = request.body.data.Uid;
+    const milestoneId = request.body.data.MilestoneId;
+    const taskId = request.body.data.TaskId;
+    const orgDomain = request.body.data.OrgDomain;
+    const date = request.body.data.Date;
+    const time = request.body.data.Time;
+    let result;
+    let status = 200;
+    console.log("my data", request.body.data)
```
Comment (Pendyala-thanvi) [inline functions/model/milestone/tark/addTask.js:46]: same here

File: src/app/services/milestone-service.service.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  taskData: Tasks[];
+  completedTasks: any[] = [];
+  incompleteTasks: any[] = [];
+  private taskDataState: Subject<boolean> = new Subject<boolean>();
+  public taskDataStateObservable = this.taskDataState.asObservable();
+  constructor(private functions: AngularFireFunctions) { }
+
+
+  getTasks(orgDomain, milestoneId) {
+    console.log("get tasks in service");
```
Comment (Pendyala-thanvi) [inline src/app/services/milestone-service.service.ts:20]: Remove here too

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**